main.go

```diff
@@ -34,12 +34,16 @@ func scoopHome() (res string) {
 			configHome = home + "\\.config"
 		}
 
-		path := configHome + "\\scoop\\config.json"
-		if content, err := os.ReadFile(path); err == nil {
+		configPath := configHome + "\\scoop\\config.json"
+		if content, err := os.ReadFile(configPath); err == nil {
 			var parser fastjson.Parser
 			config, _ := parser.ParseBytes(content)
 			res = string(config.GetStringBytes("root_path"))
-		} else {
+		}
+
+		// installing with default directory doesn't have `SCOOP`
+		// and `root_path` either
+		if len(res) == 0 {
```

Comment:
Any reason you don't just check the underlying string? ie
```suggestion
		if res == "" {
```

Reply:
No reason. I got this through Bing. I'm new to Go so I'm not familiar with Go's practice. Will apply your suggestion. Thanks for the review!